experimental/examples/bubblebench/common/src/types.ts

```diff
@@ -46,10 +46,12 @@ export function makeBubble(stageWidth: number, stageHeight: number) {
     };
 }
 
-export const makeClient = (stageWidth: number, stageHeight: number, numBubbles: number) => ({
+export const makeClient = (stageWidth: number, stageHeight: number, numBubbles: number): IClient => ({
     clientId: "pending",
     color: randomColor(),
-    bubbles: new Array(numBubbles)
+    bubbles: Array.from({length: numBubbles})
+        // False positive
+        // eslint-disable-next-line unicorn/no-useless-undefined
```

Comment:
Can we just remove the `.fill()`?

<img width="335" alt="image" src="https://user-images.githubusercontent.com/716334/207203513-7bf0d58d-c9fb-46d7-863f-4efc82580440.png">

Reply:
Oh yeah, good call 🤦‍♂️